frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -555,19 +555,18 @@ function Cell({
         ...style,
         ...(isSubtotal ? { backgroundColor: PIVOT_BG_DARK } : {}),
       }}
-      className={cx("flex-full flex-basis-none", className, {
-        "text-bold": isSubtotal,
-        "cursor-pointer": onClick,
-      })}
+      className={cx(
+        "shrink-below-content-size flex-full flex-basis-none",
+        className,
+        {
+          "text-bold": isSubtotal,
+          "cursor-pointer": onClick,
+        },
+      )}
       onClick={onClick}
     >
       <div className={cx("px1 flex align-center", { "justify-end": isBody })}>
-        {isBody ? (
-          // Ellipsified isn't really needed for body cells. Avoiding it helps performance.
```

Comment:
I don't think the performance hit will be too large. I removed it among a bunch of other tweaks a while back.